### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by GhislainMitahi**
DESKTOP VERSION

- adapt the index on the desktop version
- add a media query to make  the desktop version
- add a background image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,11 @@
             </p>
             <nav class="nav">
                 <i class="fa-solid fa-bars"></i>
+                <ul class="menu">
+                    <li><a href="http//">Portofolio</a></li>
+                    <li><a href="http//">About me</a></li>
+                    <li><a href="http//">Contact</a></li>
+                </ul>
```
Comment (zhadier) [inline index.html:23]: - [ ] Great work implementing the navbar llinks 👍. I think there is a slight discrepancy when it comes to the design though in the `font-family` and the `font-weight`. I've added an image below for comparison, kindly change it to mimic the Figma design
> Your Design
image

> Figma Design
image

File: style.css [hunk 2]
```diff
@@ -470,3 +491,144 @@ section .contact_forms .e_mail {
...
 }
+
+@media (min-width: 800px) {
+  body {
+    background-image: url(assets/background.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-origin: content-box;
+    background-size: cover;
+  }
```
Comment (zhadier) [inline style.css:502]: - [ ] [_Optional_] I think you should also explore `background-size: 100% 100%` and see how it turns out. The problem with cover is that the background starts disappearing when it's too low or too high a screen width, but this is entirely upto you and merely a suggestion.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @GhislainMitahi  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description
✔️ You have done well matching your design to the original Figma design
✔️ You have used CSS grid and flexbox well for aligning the elements
✔️ Used the font and colors defined for the template

## Required Changes ♻️
- [ ] I noticed that there doesn't seem to be any button selector effects applied so clicking and hovering over the button doesn't seem very interactive. The Figma template also defined styles for button interaction as you can see below, Kindly do implement these effects aswell.
> Button Interactions
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — APPROVED** [review]
Hi @GhislainMitahi  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters are passing ✔️ 
- clean codes 👍 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**